src/main/resources/graphql/schema/user.graphqls

```diff
@@ -15,12 +16,14 @@ input UserLoginDTO {
 }
 
 extend type Mutation {
-    createUser(user: UserDTO !): User !
+    createUserBand(user: UserDTO !, band: BandDTO !): User !
+    createUserVenue(user: UserDTO !, venue: VenueDTO !): User !
 }
 
 input UserDTO {
     username: String !
     password: String !
-    role: String !
     email: String !
+    band: BandDTO
+    venue: VenueDTO
```

Comment:
I don't think these fields are needed. `input UserDTO` is used only to describe data that we _send_, we already provide `BandDTO/VenueDTO` as a second argument for `createUserBand/createUserVenue`